include/Game/Actors/Poster/Poster.h

```diff
@@ -0,0 +1,30 @@
+#pragma once
+
+#include "Game/al/LiveActor/LiveActor.h"
+#include "Game/al/Util.h"
+#include "sead.h"
+#include "Game/Actors/Poster/PosterFall.h"
+
+class Poster : public al::LiveActor {
```

Comment:
Due to the order of the game's code, Poster seems to belong to a library called Game/MapObj. Please change that